### PR TITLE
Volunteer's organization changed from none and organization list added in edit profile

### PR DESCRIPTION
**Pull request by anjali-dhanuka**
# Description
Previously, when the user wanted to edit his profile he was not shown organization list and his organization was set to none. Organization list is now passed as a context data in form to show organizations list

Issue #581, #582 

# Type of Change:
- Code

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?
image
image

# Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by rohan033** [comment]
@anjali-dhanuka I have opened a similar PR #582. Please review the same.

**Comment by anjali-dhanuka** [comment]
@tapasweni-pathak @tonythomas01 Please review.

**Comment by tonythomas01** [comment]
-1. Needs rebase.

**Comment by anjali-dhanuka** [comment]
@tonythomas01 Done with rebasing. Please review.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,47 +1,11 @@
...
-* Subscribe to our [systers-dev](http://systers.org/mailman/listinfo/systers-dev) mailing list and send out an introductory email.
-* You can join our [slack channel](http://systers.io/slack-systers-opensource/).  Each active repo has its own channel to direct questions to (for example #powerup or #portal).  Remember that this is an inclusive community, committed to creating a safe, positive environment.  See the full [Code of Conduct](https://github.com/systers/vms/blob/master/code_of_conduct.md).
-
-[General Guidelines](#general-guidelines)  
-[Opening a New Issue](#opening-a-new-issue)  
-[Important Reminders](#important-reminders)  
-[Communicate](#communicate)  
-[Don’t Know Code?](#dont-know-code)  
+* You can join our [slack channel](http://systers.io/slack-systers-opensource/).  Each active repo has its own channel to direct questions to (for example #powerup or #portal).  
+* Remember that this is an inclusive community, committed to creating a safe, positive environment.  See the full [Code of Conduct](systers.io/code-of-conduct.html).
```
Comment (tonythomas01) [inline CONTRIBUTING.md]: why are these still here ?

Comment (tapasweni-pathak) [reply CONTRIBUTING.md]: Could you share why you feel this shouldn't be here?

Comment (tonythomas01) [reply CONTRIBUTING.md]: @tapasweni-pathak got me wrong. why are these still here in the sense - these are not part of the commit. Possibly due to rebase issues. Check the PR title. `Volunteer's organization changed from none and organization list added in edit profile`

**Comment by anjali-dhanuka** [comment]
@tapasweni-pathak @tonythomas01 Done with rebasing and I have removed other's commits too from my pr. Please review.

**Comment by anjali-dhanuka** [comment]
This functionality has been implemented in https://github.com/systers/vms/issues/581. Closing it

**Pull request closed without merging**